sources/packages/api/src/route-controllers/institution/models/institution.dto.ts

```diff
@@ -67,3 +69,10 @@ export class CreateInstitutionDto {
   @IsNotEmpty()
   postalCode: string;
 }
+
+export class InstituteDto extends PartialType(CreateInstitutionDto) {}
+
+export interface InstitutionDetailDto {
+  institution: InstituteDto;
```

Comment:
I totally understand these both objects being together here for the current scenario, but I hope that in the future when we have a more solid idea about the UI/UX, we are going to be dealing with them separately.

Reply:
I agree.